qa/tasks/cephfs/kernel_mount.py

```diff
@@ -217,7 +218,11 @@ def read_debug_file(self, filename):
             return stdout.getvalue()
         except CommandFailedError:
```

Comment:
@vshankar The whole try-catch can be replaced with if else, with `check_status=False`, this would also eliminate the need to look into stderr.getvalue() for `no such file or directory` (ENOENT), `not a directory` (ENOTDIR) and save a lot of string searching time. What do you think?